src/configuration/routes/cache.md

```diff
@@ -2,15 +2,11 @@
 
 ## Rationale
 
-Many hosting solutions are adding an additional layer (Varnish...) to
-handle caching. Platform.sh allows you to enable HTTP caching right at
-the web server level.
+Many hosting solutions are adding an additional layer (Varnish...) to handle caching. Platform.sh supports HTTP caching right at the web server level.
 
-If you disable caching, Platform.sh serves the files that are stored in
-the application directly. For example with Drupal, this means that all
-HTTP requests will bootstrap Drupal and query the database. When the
-cache is enabled, if the page has been stored in the Nginx cache, it
-won't access Drupal.
+By default, the router will respect whatever cache headers are sent by the application (either from your application code or from the `.platform.app.yaml` configuration), but the presence of any cookie headers will cause the cache to be bypassed.
+
+If you disable caching, Platform.sh serves the files that are stored in the application directly. For example with Drupal, this means that all HTTP requests will bootstrap Drupal and query the database. When the cache is enabled, if the page has been stored in the Nginx cache, it won't access Drupal.
```

Comment:
re: "all HTTP requests" - This would only be the case for requests that hit Drupal.  Static files (which could also be cached at the router) would not cause a Drupal bootstrap but would be served the webserver in the app container.